pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/ComponentBlockDef.java

```diff
@@ -26,6 +26,9 @@
     @SerializedName("hideNumber")
     private boolean hideNumber;
 
+    @SerializedName("revisionId")
+    private long revisionId;
```

Comment:
Note that these definition objects are what we use to deserialize study-builder activity configurations in order to insert them into the database (check out `ActivityBuilder` in study-builder codebase). `revisionId` is not something we'll deserialize from the activity configuration JSON. How about we change the following so it doesn't play a part in JSON?

```
private transient long revisionId;
```